### DISCUSSION TITLE
Yanky Telescope Error

### DISCUSSION THREAD

**Discussion by thecontinium** [LazyVim/LazyVim#3943] [Q&A]
Hi - simply added yanky as an extra and get the following error when I try to open telescope for yanky with <Leader>p.

```
E5108: Error executing lua: ...lazy/LazyVim/lua/lazyvim/plugins/extras/coding/yanky.lua:14: attempt to index field 'picker' (a nil value)
stack traceback:
	...lazy/LazyVim/lua/lazyvim/plugins/extras/coding/yanky.lua:14: in function <...lazy/LazyVim/lua/lazyvim/plugins/extras/coding/yanky.lua:13>
E5108: Error executing lua [string ":lua"]:1: attempt to index field 'picker' (a nil value)
```

Anybody have this ?

Thanks.

**Comment by folke** [comment]
Can't reproduce this. Are you sure everything is up to date?

**Comment by thecontinium** [comment]
Clean install works - must be something to do with my setup - I'll check.

**Comment by thecontinium** [comment]
Hi - I define my own version of telescope - I do not use Extras for telescope.
In my options I set `vim.g.lazyvim_picker = "telescope"` 
Is there anything else I need to do ?

**Reply by dpetka2001** [reply]
What do you mean your own version of Telescope? Telescope Extra is enabled by default in LazyVim unless you replace it with the `fzf-lua` Extra. Seems like you're using LazyVim in a custom setup of your own. If that's the case then change the mappings for `yanky` to use `Telescope` commands instead of the `LazyVim.pick` auto-picker.

**Reply by thecontinium** [reply]
Yes it is a custom setup. I was hoping to use extras yanky out of the box but I can see I can't now - it's tied to the lazyvim picker.
I'll configure it myself.
Thanks.

**Comment by thecontinium** [comment]
Thanks
